src/seeds.settings.cpp

```diff
@@ -52,6 +52,39 @@ void settings::reset() {
   configure(name("maxvouch"), 50);
 
 
+  // =====================================
```

Comment:
remove this file from commit, these are unrelated changes that have already been merged